web/src/people.ts

```diff
@@ -1242,9 +1252,19 @@ export function filter_people_by_search_terms(
 }
 
 export const is_valid_full_name_and_user_id = (full_name: string, user_id: number): boolean => {
+    /*
+        This function is currently only used for checking
+        the mention syntax during markdown parsing. Since
+        we do not want to parse inaccessible users as
+        mention pill, this function returns false for
+        inaccessible users. We would need to update this
+        if we would want to use this function for other
+        cases where we might want to display inaccessible
+        users as "Unknown user".
+    */
```

Comment:
We also not show the mention pill if user types `@**Unknown user|{user_id}**` when `user_id` is ID of an inaccessible user.